dosbox-staging/dosbox-staging
# Add countries and historic locales from OS/2 Warp 4.52

# Description

Added country codes supported by OS/2, added historic locale definitions from OS/2 for countries that didn't have them. Thanks to @rderooy for collecting the data.

## Related issues

https://github.com/dosbox-staging/dosbox-staging/issues/3128


# Manual testing

- execute `dosbox --list-countries` and check if the newly added countries are visible
- try to set new country codes in `country` config setting

# Checklist

I have:
